### DISCUSSION TITLE
Postal API + Mautic ( Symfony )

### DISCUSSION THREAD

**Discussion by k040711** [postalserver/postal#2463] [Help with using Postal]
Is there anyone here using symfony / mautic with postal mail server?

I tried with SMTP and it works, but Mautic's sending rate is too slow because it has to make a "handshake" with the SMTP connection

Is there perhaps a symfony / mautic plugin for Postal ?

I was using one for mailgun before I switched and with the API it was blazing fast

Or is there perhaps some developers here experienced with symfony that can build a Mautic plugin to transport emails by HTTP post to Postal?

I realize this is way outside the scope of normal help questions but figured I'd ask anyways in case someone here already got a solution.

tnx

**Comment by IonutOjicaDE** [comment]
Do you know about this plugin?
https://github.com/LucasCCS/postal_server_api_transport_for_mautic

**Comment by fanpero87** [comment]
I installed the Plugin on my `Mautic 4.4.12` server.
On the Email settings I now see a `Postal - API` option. 
I made an API key under the credentials option on Postal and I'm now sending emails via API.

I still haven't figure out how to set up bounce emails.
On Postal I made a web hook like this:
URL: `https://mauticserver.com/mailer/postal_api/callback`
Enabled: `Yes - send request to this webhook`
Events: checked `MessageDeliveryFailed and MessageBounced`

From Mautic when I send an email to for example `fakeemailaddress@hotmail.com`, on Postal I just got a `HARDFAIL` but not bounce and the email doesnt get marked as `bounced`.

How can I setup bounce emails with Postal and Mautic?
